script.go

```diff
@@ -219,6 +219,11 @@ func (c *Compiled) RunContext(ctx context.Context) (err error) {
 	v := NewVM(c.bytecode, c.globals, c.maxAllocs)
 	ch := make(chan error, 1)
 	go func() {
+		defer func() {
+			if r, ok := recover().(error); ok {
```

Comment:
I guess the recovered value doesn't have to be `error` right? We can wrap it as error if it's not an `error` object.

Reply:
I've changed it to look for string and error with a type switch, defaulting to a prefix of "unknown panic:" if the type is unknown.